meson.build

```diff
@@ -721,6 +721,7 @@ if fs_search_path_extra != ''
 endif
 conf.set_quoted('FS_SEARCH_PATH', fs_search_path)
 
+systemdsystemunitdir = ''
```

Comment:
The use of the directory should be everywhere covered by `if systemd.found()`. (It does not mean that the patch is incorrect :-).